src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import { getCase, uploadCase } from '../../../../../supabase/queries/case';
+import styles from './styles';
+
+enum caseStatus {
+  NotStarted,
+  InProgress,
+  Settled,
+  Cancelled,
+}
+
+export interface Case {
+  id: number;
+  approved: boolean;
+  title: string;
+  summary: string;
+  image: string;
+  caseStatus: caseStatus;
+}
+
+function AddCase() {
+  const local = useLocalSearchParams();
+  const { caseId } = local;
+  const router = useRouter();
+
+  const [dataImage, setDataImage] = useState('');
```

Comment:
We also don't need these state variables because we can handle this in the previous screen